### PR TITLE
[Azure AD roles] Assign roles to groups GA

### PR DESCRIPTION
**Pull request by rolyon**

### CODE REVIEW

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+      {
+        "ChangeList": [
+            {
+                "Id": "3f30920e-41d4-4a40-b4dc-f95070985bfa",
+                "ApiChange": "Property",
+                "ChangedApiName": "isAssignableToRole",
+                "ChangeType": "Addition",
+                "Description": "Added the **isAssignableToRole** property to the Create group API.",
```
Comment (abhijeetsinha) [inline changelog/Microsoft.DirectoryServices.json:10]: @rolyon  - We add attributes to a resource type, not an API. isAssignabletToRole attribute is added to Group object. 
"Added the **isAssignableToRole** property to the groups resource type."

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+                "Target": "isAssignableToRole"
+            }
+        ],
+        "Id": "3f30920e-41d4-4a40-b4dc-f95070985bfa",
+        "Cloud": "prd",
+        "Version": "v1.0",
+        "CreatedDateTime": "2020-03-19T00:00:00Z",
+        "WorkloadArea": "Groups",
```
Comment (abhijeetsinha) [inline changelog/Microsoft.DirectoryServices.json:18]: @rolyon  - Workload should be "Identity and access"

File: changelog/Microsoft.DirectoryServices.json [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+                "ChangedApiName": "isAssignableToRole",
+                "ChangeType": "Addition",
+                "Description": "Added the **isAssignableToRole** property to the Create group API.",
+                "Target": "isAssignableToRole"
+            }
+        ],
+        "Id": "3f30920e-41d4-4a40-b4dc-f95070985bfa",
+        "Cloud": "prd",
+        "Version": "v1.0",
+        "CreatedDateTime": "2020-03-19T00:00:00Z",
```
Comment (abhijeetsinha) [inline changelog/Microsoft.DirectoryServices.json:17]: @rolyon  - Created date has 2020 as year

**Comment by rolyon** [comment]
#sign-off

**Pull request merged**